src/modules/Providers/ProvidersPage.tsx

```diff
@@ -141,31 +142,29 @@ export const ProvidersPage = ({ namespace, kind: reference }: ResourceConsolePag
   );
 };
 
-const Page = ({
-  dataSource,
-  namespace,
-  title,
-  userSettings,
-}: {
+const Page: React.FC<{
   dataSource: [MergedProvider[], boolean, boolean];
   namespace: string;
   title: string;
   userSettings: UserSettings;
-}) => (
-  <StandardPage<MergedProvider>
-    addButton={<AddProviderButton />}
-    dataSource={dataSource}
-    RowMapper={ProviderRow}
-    fieldsMetadata={fieldsMetadata}
-    namespace={namespace}
-    title={title}
-    userSettings={userSettings}
-  />
-);
+}> = ({ dataSource, namespace, title, userSettings }) =>
+  (dataSource[0]?.length ?? 0) <= 0 ? (
```

Comment:
this will need to be more complex, maybe even exxtracted into a const ( `const showEmptyState =` .... `? ), because if CNV is installed and the MTV operator found it, it will create a default provider and we will need to ignore it, and show empty state even if we have this automatically created provider.

Reply:
to see the default provider, you need to install CNV + MTV, an automatically created providers should be created.

Reply:
hmm, if we hide the automatically created provider, what if users will try to create it thinking it's missing? ... 
so, ok, lets leave the logic as is

Reply:
btw/ `.length` is `unsigned int`  the <= is a bit extreme :-)

Reply:
To continue splitting hairs, `Array.length` is a `number` and could in theory be negative.  Plus javascript doesn't have unsigned numbers!  And never assume there isn't a difference between `+0` and `-0`!!!

I'll tweak the logic with a var to make it easier to modify going forward.  I'd rather keep the current `=== 0` check at least for now.  I'm not sure if there is a definitive way to determine if the single provider is the default one installed with CNV+Forklift together on the same cluster.